composables/content-render.ts

```diff
@@ -4,6 +4,7 @@ import { Fragment, h, isVNode } from 'vue'
 import type { VNode } from 'vue'
 import { RouterLink } from 'vue-router'
 import { decode } from 'tiny-decode'
+import { find as findEmoji } from 'node-emoji'
```

Comment:
The only thing I am not sure is the introduction of this dep. This module supports to work on client side, and this package seems to be designed for node. I guess we already have some logic to extract emoji from a string that we could reuse.

Reply:
I also thought that, but in the end `node-emoji` is "just" a wrapper around `emojilib`, that is also used by `emoji-mart`
However, `emoji-mart` has a `getEmojiDataFromNative`, but this is async and also need to call `init` once somewhere before.
But `emoji-mart` is just the emoji-picker component and also I'm not sure if it is emoji-"complete"

all in all, we need somehow a lookup set to "translate" emojis into text, and we could do this manually by hand (feels overkill and very cumbersome) or we just use a lib like `node-emoji` (wrapper around `emojilib`)

I feel safe about `node-emoji` and I assume it just has node in its name, because it is used for nodejs/js 🤷

Reply:
If we could ensure only one `emojilib` gets bundled, then I am okay with having it.

Reply:
Just checked Netlify, against the main branch, the full bundle size is

- this PR: `8.78 MB (1.73 MB gzip)`
- main: `8.45 MB (1.68 MB gzip)`

And the `entry.js`

- this PR: `763.66 kB │ gzip: 218.47 kB`
- main: `537.79 kB │ gzip: 168.74 kB`

I think I am ok with the full bundle adding 330KB, but it shouldn't go to the main entry. `(763 - 537) / 537 = 42%` increase of the initial entry bundle.

I guess we need to somehow make it lazy-loaded.